app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{#link-to "rentals.show" rentalUnit.slug}}
```

Comment:
I'm not finding an easy way to do this as we essentially have to make our component aware of our parent.  I had a solution with bubbling actions and preventing the default action when the target is the image, but that adds a lot of unnecessary complexity to something that is supposed to be simple.

The easiest solution may just be adding a link to the component saying "View Details" or something.